release/jobs/health_monitor/templates/health_monitor.yml.erb

```diff
@@ -141,6 +141,12 @@ plugins:
       <% end %>
   <% end %>
 
+   <% if p("hm.syslog_event_forwarder_enabled", false) %>
+  - name: syslog_event_forwarder
+    events:
+      - alert
```

Comment:
@mrdavidlaing Sure, we can add a setting with which you can also forward heartbeat events. We decided not to do this in that change, because were not sure if syslog is the right format for these types of events. Instead, we use the [graphite-plugin](https://github.com/cloudfoundry/bosh/blob/master/bosh-monitor/lib/bosh/monitor/plugins/graphite.rb) to forward them to a stream processor and time-series database and actually use the data provided in the heartbeats for alerting and dashboards.